basics.tex

```diff
@@ -222,6 +222,12 @@ \section{Formatting Strings}\label{s:basics-formatting}
 color is blue and capitalized is BLUE
 \end{minted}
 
+This allows us to succinctly add variables to a string instead of
```

Comment:
I usually end lines with a colon ':' if they are introducing a code block like this

Reply:
Other than that, good to merge - thanks.